src/ekstat.erl

```diff
@@ -1,14 +1,20 @@
 -module(ekstat).
 
 %% API
--export([open/0, update/1, clear/1, read/1, read/2, read/3, read/4, read/5, read/6]).
+-export([open/0, close/1, clear/1, update/1, read/1, read/2, read/3, read/4, read/5, read/6]).
```

Comment:
Line 4 is too long: -export([open/0, close/1, clear/1, update/1, read/1, read/2, read/3, read/4, read/5, read/6])..